diff --git a/node_modules/react-lines-ellipsis/lib/index.js b/node_modules/react-lines-ellipsis/lib/index.js
--- a/node_modules/react-lines-ellipsis/lib/index.js
+++ b/node_modules/react-lines-ellipsis/lib/index.js
@@ -141,7 +141,7 @@
       this.copyStyleToCanvas();
     }
 
-    if (this.props !== prevProps) {
+    if (usedProps.some(key => this.props[key] !== prevProps[key])) {
       this.reflow(this.props);
     }
   };
diff --git a/node_modules/react-lines-ellipsis/lib/index.modern.mjs b/node_modules/react-lines-ellipsis/lib/index.modern.mjs
--- a/node_modules/react-lines-ellipsis/lib/index.modern.mjs
+++ b/node_modules/react-lines-ellipsis/lib/index.modern.mjs
@@ -117,7 +117,7 @@
       this.copyStyleToCanvas();
     }
 
-    if (this.props !== prevProps) {
+    if (usedProps.some(key => this.props[key] !== prevProps[key])) {
       this.reflow(this.props);
     }
   }
