diff --git a/node_modules/zod-gpt/dist/src/completion.js b/node_modules/zod-gpt/dist/src/completion.js
index a8d7b08..4d74c0b 100644
--- a/node_modules/zod-gpt/dist/src/completion.js
+++ b/node_modules/zod-gpt/dist/src/completion.js
@@ -43,7 +43,6 @@ async function chat(model, messages, _opt) {
     utils_1.debug.log('⬆️ sending request:', messages);
     try {
         const hasFunctionCall = !(model instanceof llm_api_1.AnthropicChatApi ||
-            model instanceof llm_api_1.AnthropicBedrockChatApi ||
             model instanceof llm_api_1.GroqChatApi);
         const schemaInstructions = !hasFunctionCall && _opt?.schema && JSON.stringify(jsonSchema);
         const firstSchemaKey = !hasFunctionCall &&
